### DISCUSSION TITLE
Primeiro teste do app web com a api

### DISCUSSION THREAD

**Discussion by tiaotj** [balta-io/forum#712] [Tira Dúvidas] [answered]
Curso FullStack .NET
Aula: Login Logout

Quando tentei realizar o primeiro teste com a Api, deu erro, quando aperto f12 aparece um monte de erro

![image](https://github.com/user-attachments/assets/d8bf433a-f9ce-4f5c-9515-1e0a2429f5fc)

Não consegui identificar o problema, o Swagger tá rodando normal, e consigo fazer o registro por lá

**Comment by andrebaltieri** [comment]
Bom dia, @tiaotj, como vai?

Este é um problema de CORS, eu mostro como resolver isto no curso!
https://balta.io/player/assistir/c4bfdf26-8709-46f4-b918-d59e78953e4a/524892aa-f6db-4741-872e-8b00115fb79e

**Comment by tiaotj** [comment]
Meu código está exatamente o mesmo da aula, e mesmo assim não funciona
![image](https://github.com/user-attachments/assets/7ed70854-b7e0-4285-969e-670b2b00f03a)
![image](https://github.com/user-attachments/assets/a6030714-80f9-4ede-86bc-5f0b3595cf9e)
![image](https://github.com/user-attachments/assets/618d584c-436a-43c3-a585-56f41ef27a08)
![image](https://github.com/user-attachments/assets/84c8c5bd-05f8-4e48-a8ae-f2a97f2fb5af)
![image](https://github.com/user-attachments/assets/2d2b1ef4-ea42-42f0-874f-d0b69c79bb9d)
![image](https://github.com/user-attachments/assets/c020332b-363d-432e-9d3e-0acca497365a)

**Reply by andrebaltieri** [reply]
@tiaotj mas sua aplicação está executando nas mesmas portas? Pois as portas são aleatórias... provavelmente as suas foram gerados números diferentes dos meus!

**Reply by tiaotj** [reply]
Está com essa configuração
  "FrontEndUrl": "http://localhost:5266",
  "BackEndUrl": "http://localhost:5144/",

**Reply by tiaotj** [reply]
Subi meu código
https://github.com/tiaotj/CursoFullStack.git

**Comment by tiaotj** [comment, marked as answer]
Consegui resolver, era um conflito de url entre o configuration do .web, e as url do cors.
E o ponto principal é que quando eu executava o front end pelo dotnet run, ele rodava em uma porta, e quando executava pelo player do vs2022 ele rodava em outra porta, aí o cors não deixava rodar.

**Reply by andrebaltieri** [reply]
Showwww!!!
